point-of-sale-app/inventory/src/main/java/com/google/abmedge/inventory/InventoryController.java

```diff
@@ -120,6 +120,20 @@ public String readiness() {
     return new ResponseEntity<>(jsonString, HttpStatus.OK);
   }
 
+  @PostMapping(value = "/items_by_id",
+      consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
+  public ResponseEntity<String> items(@RequestBody List<String> idList) {
+    List<Item> inventoryItems = new ArrayList<>();
```

Comment:
Why do you need to explicitly initialize inventoryItems? Let's see, do you have multiple items with same UUID key and needs enumeration?

Reply:
Not necessarily. I've added a `collect` instead of explicit instantiation in bfde804